public/example_templates/netjsonmap-indoormap.html

```diff
@@ -120,37 +121,31 @@
           },
 
           onReady: function presentIndoormap() {
-            const netjsonmap = this.leaflet;
+            const netjsonmap = this.leaflet; 
```

Comment:
```suggestion
            const netjsonmap = this.leaflet;
```

There was a blank space at the end of line. You can configure your code editor to automatically remove them upon save.

Reply:
Updated!